connectors/class-connector-acf.php

```diff
@@ -147,13 +149,219 @@ public function action_links( $links, $record ) {
+					'option_value' => $value,
+				),
+				$post_id,
+				'options',
+				'updated'
+			);
+		} elseif ( 'layout' === $property ) {
+			if ( 'deleted' === $action ) {
+				return;
+			}
+
+			$options = array(
+				'no_box'  => esc_html_x( 'Seamless (no metabox)', 'acf', 'stream' ),
+				'default' => esc_html_x( 'Standard (WP metabox)', 'acf', 'stream' ),
+			);
+
+			$this->log(
+				/* translators: %1$s: form title, %2$s a layout (e.g. "Contact", "Seamless") */
+				esc_html_x( 'Style of "%1$s" updated to "%2$s"', 'acf', 'stream' ),
+				array(
+					'title'        => $post->post_title,
+					'option_label' => $options[ $value ],
+					'option'       => $property,
+					'option_value' => $value,
+				),
+				$post_id,
+				'options',
+				'updated'
+			);
+		} elseif ( 'hide_on_screen' === $property ) {
+			if ( 'deleted' === $action ) {
+				return;
+			}
+
+			$options = array(
+				'permalink'       => esc_html_x( 'Permalink', 'acf', 'stream' ),
+				'the_content'     => esc_html_x( 'Content Editor', 'acf', 'stream' ),
+				'excerpt'         => esc_html_x( 'Excerpt', 'acf', 'stream' ),
+				'custom_fields'   => esc_html_x( 'Custom Fields', 'acf', 'stream' ),
+				'discussion'      => esc_html_x( 'Discussion', 'acf', 'stream' ),
+				'comments'        => esc_html_x( 'Comments', 'acf', 'stream' ),
+				'revisions'       => esc_html_x( 'Revisions', 'acf', 'stream' ),
+				'slug'            => esc_html_x( 'Slug', 'acf', 'stream' ),
+				'author'          => esc_html_x( 'Author', 'acf', 'stream' ),
+				'format'          => esc_html_x( 'Format', 'acf', 'stream' ),
+				'featured_image'  => esc_html_x( 'Featured Image', 'acf', 'stream' ),
+				'categories'      => esc_html_x( 'Categories', 'acf', 'stream' ),
+				'tags'            => esc_html_x( 'Tags', 'acf', 'stream' ),
+				'send-trackbacks' => esc_html_x( 'Send Trackbacks', 'acf', 'stream' ),
+			);
+
+			if ( is_array( $value ) && count( $options ) === count( $value ) ) {
+				$options_label = esc_html_x( 'All screens', 'acf', 'stream' );
+			} elseif ( empty( $value ) ) {
+				$options_label = esc_html_x( 'No screens', 'acf', 'stream' );
+			} else {
+				$options_label = implode( ', ', array_intersect_key( $options, array_flip( $value ) ) );
+			}
+
+			$this->log(
+				/* translators: %1$s: a form title, %2$s: a display option (e.g. "Contact", "All screens") */
+				esc_html_x( '"%1$s" set to display on "%2$s"', 'acf', 'stream' ),
+				array(
+					'title'        => $post->post_title,
+					'option_label' => $options_label,
+					'option'       => $property,
+					'option_value' => $value,
+				),
+				$post_id,
+				'options',
+				'updated'
+			);
+		}
+	}
+
 	/**
 	 * Track addition of post meta
 	 *
 	 * @action added_post_meta
 	 */
 	public function callback_added_post_meta() {
-		call_user_func_array( array( $this, 'check_meta' ), array_merge( array( 'post', 'added' ), func_get_args() ) );
+		$this->check_meta( 'post', 'added', ...func_get_args() );
```

Comment:
@kidunot89 what is the PHP version compatibility of the spread operator?

Reply:
:man_facepalming: 7.4, my bad :sweat_smile:.

Reply:
@kopepasah Upon doing further research, I learned that this is called **Argument Unpacking** in PHP, and yes it was implemented in PHP 5.6. Official RFC [here](https://wiki.php.net/rfc/argument_unpacking)

Reply:
Thanks for the additional research! I was not aware it had been around for so long. 😱